File: translation_output/w_test/wizardcoder/codenet/python/java/p02694.java
Below is an instruction that describes a task. Write a response that appropriately completes the request.

### Instruction:
Translate the following python code to java and enclose your solution inside ```java```.
A sample test case is provided below:

Test input:
103

Expected output:
3


```
import math
x = int(input())

y = 100
ans = 0
while y < x:
    ans += 1
    y = math.floor(y*1.01)

print(ans)
```

### Response:
```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int x = sc.nextInt();

        int y = 100;
        int ans = 0;
        while (y < x) {
            ans++;
            y = (int) Math.floor(y * 1.01);
        }

        System.out.println(ans);
    }
}
```<|endoftext|>

File: translation_output/w_test/wizardcoder/codenet/java/python/p02694.py
Below is an instruction that describes a task. Write a response that appropriately completes the request.

### Instruction:
Translate the following java code to python and enclose your solution inside ```python```.
A sample test case is provided below:

Test input:
103

Expected output:
3


```
import java.util.Scanner;

class p02694 {

    public static void main(String[] args) {
        Scanner stdIn=new Scanner(System.in);

        long x=stdIn.nextLong();
        int count=0;

        long a=100;
        while(true){
            if(a>=x){
                break;
            }
            a=a+a/100;
            count++;
        }
        System.out.println(count);
        
    }

}
```

### Response:
```python
import java.util.Scanner;

class p02694:

    def main(self):
        stdIn = Scanner(System.in)

        x = stdIn.nextLong()
        count = 0

        a = 100
        while True:
            if a >= x:
                break
            a = a + a/100
            count += 1

        System.out.println(count)

p02694().main()
```<|endoftext|>
